src/helper.ts

```diff
@@ -15,11 +15,11 @@
  * limitations under the License.
  */
 
-import * as crypto from 'crypto';
+import crypto from 'crypto';
```

Comment:
How does this work? Reading [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import), this syntax means "import default", while we need "import everything as a namespace".